### PR TITLE
0.43 Filter minican names

### PR DESCRIPTION
**Pull request by FedericoCeratto**

### CODE REVIEW

File: af/fastpath/fastpath/core.py [hunk 1]
```diff
@@ -89,24 +89,18 @@ def setup():
...
+    ap.add_argument("--update", action="store_true", help=h)
+    h = "Stop after feeding N measurements"
+    ap.add_argument("--stop-after", type=int, help=h, default=None)
+    h = "Do not insert measurement in database"
+    ap.add_argument("--no-write-to-db", action="store_true", help=h)
+    h = "Keep files downloaded from S3 in the local cache"
+    ap.add_argument("--keep-s3-cache", action="store_true", help=h)
+    ap.add_argument("--ccs", help="Filter comma-separated CCs when feeding from S3")
+    h = "Filter comma-separated test names when feeding from S3 (without underscores)"
+    ap.add_argument("--testnames", help=h)
```
Comment (hellais) [inline af/fastpath/fastpath/core.py:101]: Perhaps it's useful to add a docstring to the testnames argument mentioning that the test_name needs to be without the `_` (this is the case right?).

**Review by hellais — APPROVED** [review]
Left a minor comment for an improvements. Otherwise looks good to me.

File: af/fastpath/fastpath/core.py (hunk 1 above)
Comment (FedericoCeratto) [reply af/fastpath/fastpath/core.py:101]: It's written in the help message.

Comment (FedericoCeratto) [reply af/fastpath/fastpath/core.py:101]: Added to the function's docstring.

**Comment by FedericoCeratto** [comment]
Deployed to test and prod.

**Review by hellais — APPROVED** [review]
Yes this looks good to merge.

**Pull request merged**